### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by rotsen18**

### CODE REVIEW

**Review by KatiukhaO — APPROVED** [review]
intresting way, $5 - 7 lines

**Comment by rotsen18** [comment]
DRY;) thanks for review

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+    def __add__(self, other):
+        return Distance(self.km + self.get_other_km(other))
+
+    def __iadd__(self, other):
+        self.km += self.get_other_km(other)
+        return self
+
+    def __mul__(self, other):
+        self.km *= other
+        return self
```
Comment (TPolina) [inline app/main.py:24]: in this method, you should return other Distance object and not modify given

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,46 @@
...
+
+    def __truediv__(self, other):
+        self.km = round(self.km / other, 2)
+        return self
```
Comment (TPolina) [inline app/main.py:28]: same as in `__mul__` method

**Review by TPolina — CHANGES_REQUESTED** [review]

**Review by TPolina — APPROVED** [review]
Great! interesting solution

**Comment by rotsen18** [comment]
Thanks ;)

**Pull request closed without merging**